README.md

```diff
@@ -90,6 +90,15 @@ Dynamic settings include:
 * Access Permissions
 * Rate Limits
 
+|Parameter|Description|
+|---------|-----------|
+|routes|Path(s) to route to specific upstreams or to respond with configured body.|
+|route-rate|Parameters for vote endpoint.|
+|applications|A list of deployed AI DIAL Applications and their parameters:<br />`endpoint`: AI DIAL Application API for chat completions.<br />`iconUrl`: a path to the icon used for the AI DIAL Application on the UI.<br />`description`: a brief description of the AI DIAL Application rendered on the UI.<br />`displayName`: a name of the AI DIAL Application used on the UI.|
+|models|A list of deployed models and their parameters:<br />`type`: specify `chat` or `embedding` model type.<br />`iconUrl`: a path to the icon used for the model on the UI.<br />`description`: a brief description of the model rendered on the UI.<br />`displayName`: a name of the model rendered on the UI.<br />`endpoint`: model API for chat completions or embeddings.<br />`upstreams`: upstreams are used for load-balancing. A request will be sent to the configured model endpoint and will contain X-UPSTREAM-ENDPOINT and X-UPSTREAM-KEY headers:<br />`endpoint`: model endpoint.<br />`key`: your API key.|
```

Comment:
`key` is obsolete. Please remove

Reply:
But how do we pass the Azure OpenAI key now?

Reply:
Adjusted table style based on problem review